### ISSUE TITLE
Remove focus styles in YoastTabs

### ISSUE THREAD

**Issue by Xyfi** [Yoast/yoast-components#312]
Currently in YoastTabs when you click on the tab content, focus styles are applied to the content container. The content must remain focusable for a11y reasons, but the focus styles should be removed. 

### Chrome
<img width="1229" alt="title" src="https://user-images.githubusercontent.com/5389513/30953550-b893654e-a42c-11e7-8655-6fcab46d877b.png">

### Safari 11
<img width="1612" alt="title" src="https://user-images.githubusercontent.com/5389513/30953612-f350ef80-a42c-11e7-938a-0950fe672bb4.png">

### Firefox
<img width="1238" alt="title" src="https://user-images.githubusercontent.com/5389513/30953658-2821fe02-a42d-11e7-8b3c-51455feb5771.png">
